core/src/main/java/org/infinispan/topology/ClusterTopologyManagerImpl.java

```diff
@@ -546,18 +544,16 @@ private ClusterCacheStatus initCacheStatusIfAbsent(String cacheName, CacheMode c
       return cacheStatusMap.computeIfAbsent(cacheName, (name) -> {
          // We assume that any cache with partition handling configured is already defined on all the nodes
          // (including the coordinator) before it starts on any node.
-         LostDataCheck lostDataCheck = ClusterTopologyManagerImpl::distLostDataCheck;
          // TODO Partition handling config should be part of the join info
          AvailabilityStrategy availabilityStrategy;
          Configuration config = configurationManager.getConfiguration(cacheName, true);
          PartitionHandling partitionHandling =
                config != null ? config.clustering().partitionHandling().whenSplit() : null;
          boolean resolveConflictsOnMerge = resolveConflictsOnMerge(config, cacheMode);
          if (partitionHandling != null && partitionHandling != PartitionHandling.ALLOW_READ_WRITES) {
-            availabilityStrategy = new PreferConsistencyStrategy(eventLogManager, persistentUUIDManager, lostDataCheck);
+            availabilityStrategy = new PreferConsistencyStrategy(eventLogManager, persistentUUIDManager, LostDataCheck.DEFAULT);
```

Comment:
We always use the `DEFAULT` implementation, no need to pass a ctor arg.

Reply:
Well, in that case, I will remove the `LostDataCheck` interface too :+1: